dev/shiza16/modules.py

```diff
@@ -0,0 +1,256 @@
+import pandas as pd
+import numpy as np
+import matplotlib.pyplot as plt
+import seaborn as sns
+from sklearn.model_selection import train_test_split
+from sklearn.svm import SVC
+<<<<<<< HEAD
```

Comment:
Looks like you have a merge conflict in this file. Git wants you to keep either from here to line 83 or from line 85 to line 255 but not both. It looks like you meant to keep the second part. To keep the second part, delete lines 7-84 and line 256.